example/package-lock.json

```diff
@@ -0,0 +1,15638 @@
+{
```

Comment:
The example project uses `yarn` instead of `npm` so this lockfile shouldn't be needed. (Probably got it from running `npm install` vs `yarn install`

Reply:
ah, right, my bad.  Thanks for catch